src/Artsy/Relay/createRelaySSREnvironment.ts

```diff
@@ -74,6 +74,14 @@ export function createRelaySSREnvironment(config: Config = {}) {
   const network =
     relayNetwork ||
     new RelayNetworkLayer([
+      // TODO: Better introspection around if this is a SearchBar query,
+      // or further refactoring to extract `addMiddlewareToEnvironment(environment)`,
+      // to be used in the SearchBar QueryRenderer (for example).
+      next => req => {
+        if (req.variables.term === "")
```

Comment:
I have a few other things on my plate that I want to get done first, so can you file a ticket for allowing to scope this to just a single QueryRenderer and assign it to me? In the meantime you can perhaps ensure this short-circuiting only applies to the right query by adding a check for something like `req.query.includes("SearchBarQuery")`

Reply:
Will do!

Reply:
Turns out `req` isn't the usual 'express/node' request object, but a `RelayRequest`, so `req.id` is the name of the query `SearchSuggestQuery`, doesn't even need an `includes`!

Reply:
Perfect 👌